assets/weap.qrc

```diff
@@ -411,6 +411,5 @@
     <file>imgs/weap/swords/wffg_b.webp</file>
     <file>imgs/weap/swords/xpml_a.webp</file>
     <file>imgs/weap/swords/xpml_b.webp</file>
-    <file>imgs/weap/main/weap.png</file>
```

Comment:
Here again, could you point me to the instance where it has been referenced once before to necessitate the removal of what looks to you as being a repeated entry of the said file? Please and thank you.